ComponentKit/StatefulViews/CKStatefulViewComponentController.h

```diff
@@ -32,7 +32,7 @@
 
  This controller's corresponding component must subclass CKStatefulViewComponent.
  */
-@interface CKStatefulViewComponentController : CKComponentController
+@interface CKStatefulViewComponentController<__covariant ComponentType:CKComponent *> : CKComponentController
```

Comment:
Interesting. I almost wonder if this component controller should be covariant not only in the component type, but the view type... Anyway, let's deal with that later.

I'm not positive about this, but shouldn't this interface be:

```
@interface CKStatefulViewComponentController<__covariant ComponentType:CKComponent *> : CKComponentController<ComponentType>
```